docs/content/node-operation/access-node-setup.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: Setting Up a Flow Access Node
+sidebar_title: Access Node Setup
+description: How to run a Permissionless Access node
+---
+
+This guide is for running a Permissonless Access node on Flow. If you are planning to run a different type of staked node then see [node bootstrap](https://developers.flow.com/nodes/node-operation/node-bootstrap).
+
+First you'll need to provision a machine or virtual machine to run your node software. Please see follow the [node-provisioning](https://developers.flow.com/nodes/node-operation/node-provisioning) guide for it.
+
+
+## Generate keys
```

Comment:
WIP - coming up in Part2

Reply:
use relative link (node-provisioning.mdx)

Reply:
swtiched to relative links